services/repository-managers/src/main/java/org/sagebionetworks/repo/util/jrjc/JiraClientImpl.java

```diff
@@ -11,6 +12,7 @@
 import org.json.simple.parser.ParseException;
 import org.sagebionetworks.simpleHttpClient.*;
 import org.sagebionetworks.url.HttpMethod;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
You can inject the StackConfiguration with @Autowire since the manager imports it as a bean.

Additionally, you can annotate this class with the @Service annotation and remove the xml bean definition from the manager-sbp.xml since we enabled auto scanning of beans.